### ISSUE TITLE
umitools

### ISSUE THREAD

**Issue by chen5027** [ubccr/software-layer#197]
Silimarly, another module is missing for umitools

**Comment by dsajdak** [comment]
@chen5027 is this to software you're referring to?  https://umi-tools.readthedocs.io

**Comment by tonykew** [comment]
Test build of UMI-tools version 1.0.1 completed - more info to follow...

**Comment by chen5027** [comment]
Hello, Tony, I tested UMI-tools by module spider or module load but cannot find it.  This is true for all my requested modules including deeptools, picard and subread as well.  Could you show me the screenshot of your successful module loading or let me know how I should load these new modules please?

**Comment by tonykew** [comment]
Modules are tested internally before being published, hence "more info to follow..." after
my message above.

UMI-tools version 1.0.1 has now been published to the ccrsoft/2023.01 software release

```
vortex1$ module spider umi-tools

----------------------------------------------------------------------------
  umi-tools:
----------------------------------------------------------------------------
    Description:
      Tools for handling Unique Molecular Identifiers in NGS data sets

     Versions:
        umi-tools/1.0.1-Python-3.9.6
[...]
vortex1$ module spider umi-tools/1.0.1-Python-3.9.6
----------------------------------------------------------------------------
  umi-tools: umi-tools/1.0.1-Python-3.9.6
----------------------------------------------------------------------------
    Description:
      Tools for handling Unique Molecular Identifiers in NGS data sets


    You will need to load all module(s) on any one of the lines below before the "umi-tools/1.0.1-Python-3.9.6" module is available to load.

      gcc/11.2.0  openmpi/4.1.1
[...]
vortex1$ module load gcc/11.2.0 openmpi/4.1.1 umi-tools/1.0.1-Python-3.9.6
vortex1$ 
vortex1$ umi_tools --help
For full UMI-tools documentation, see: https://umi-tools.readthedocs.io/en/latest/


umi_tools.py - Tools for UMI analyses
===============================================

:Author: Tom Smith & Ian Sudbury, CGAT
:Release: $Id$
:Date: |today|
:Tags: Genomics UMI

There are 6 tools:

  - whitelist
  - extract
  - group
  - dedup
  - count
  - count_tab

To get help on a specific tool, type:

    umi_tools <tool> --help

To use a specific tool, type::

    umi_tools <tool> [tool options] [tool arguments]

vortex1$ 

```
